src/main/java/com/khadri/jdbc/prepare/data/customer/dao/CustomerInsertDao.java

```diff
@@ -2,15 +2,19 @@
 
 import java.sql.Connection;
 import java.sql.PreparedStatement;
+import java.sql.ResultSet;
 import java.sql.SQLException;
+import java.sql.Statement;
 
 import com.khadri.jdbc.prepare.connection.JdbcConnectionUtil;
 import com.khadri.jdbc.prepare.data.customer.dao.model.Customer;
 
 public class CustomerInsertDao {
```

Comment:
change class name from CustomerInsertDao to CustomerDao
add the select and insert methods into it.